File: from-3201-to-3300/3223-minimum-length-of-string-after-operations/solution.go
func minimumLength(s string) int {
    counts := make(map[byte]int)
    n := len(s)
    for i := 0; i < n; i ++ {
        counts[s[i]] += 1
    }
    toDelete := 0
    for _, v := range counts {
        if v % 2 == 0 {
            toDelete += v - 2
        } else {
            toDelete += v - 1
        }
    }
    return n - toDelete
}

File: from-3201-to-3300/3223-minimum-length-of-string-after-operations/solution.py
class Solution:
    def minimumLength(self, s: str) -> int:
        counts = Counter(s)
        to_delete = 0
        for _, v in counts.items():
            to_delete += v - 1 if v % 2 == 1 else v - 2

        return len(s) - to_delete